source/common/filter/config_discovery_impl.h

```diff
@@ -165,6 +156,40 @@ class DynamicFilterConfigProviderImpl : public DynamicFilterConfigProviderImplBa
   ThreadLocal::TypedSlot<ThreadLocalConfig> tls_;
 };
 
+// Implementation of a HTTP dynamic filter config provider.
+class HttpDynamicFilterConfigProviderImpl
```

Comment:
Missing tests for this new class.

Reply:
The code is tested by:  https://source.corp.google.com/piper///depot/google3/third_party/envoy/src/test/common/filter/config_discovery_impl_test.cc;rcl=433784789;l=74
This is the code test coverage reoport: https://storage.googleapis.com/envoy-pr/f179788/coverage/source/common/filter/config_discovery_impl.h.gcov.html